LICENSE.txt

```diff
@@ -0,0 +1,137 @@
+OPEN SOURCE LICENSE AGREEMENT
+=============================
+
+Copyright (c) 2009-2011, University of Tennessee
```

Comment:
we need to update the copyright so that 2019-2024 is columbia again.

Reply:
done